model.js

```diff
@@ -31,17 +40,19 @@ function translate(data) {
     features: []
   };
   if (list && list[0]) {
+    // Only return apartments with an "Ask" property;  this filters out geo-clusters
     const apartments = list[0].filter(node => {
       return node.Ask;
     });
-    featureCollection.features = apartments.map(formatFeature);
+
+    featureCollection.features = createFeaturesWithArcGISCompliantIDs(apartments, formatFeature)
```

Comment:
let's not bring the name ArcGIS into this example. Koop is supposed to support multiple different output formats. I'd just generalize the language here